src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/RssLink.java

```diff
@@ -11,7 +11,7 @@
 
     public RssLink(@NonNull Blog blog) {
         Preconditions.checkArgument(StringUtils.isNotBlank(blog.getRss()),
-                "Rss link can not be null");
+                "Rss link can not be empty.");
```

Comment:
The precondition message is still misleading. It should exactly reflect what is wrong with the parameter. Now it says that link cannot be empty. What about _nulls_? The message should be clear enough and reflect what is really tested here.